packages/mcp-server-mocks/src/index.ts

```diff
@@ -856,6 +856,198 @@ export const restHandlers = buildHandlers([
   },
 ]);
 
+// Add handlers for mcp.sentry.dev and localhost
+export const ragSearchHandlers = [
+  http.post("https://mcp.sentry.dev/api/rag-search", async ({ request }) => {
```

Comment:
prob will change the url here too

likely this becomes /api/search, and the chat agent stuff goes to /chat/api or similar